src/CoreBundle/DataModel/BackendTreeNodeDataModel.php

```diff
@@ -255,7 +275,7 @@ public function jsonSerialize()
     {
         $jsTreeItem = [
             'id' => $this->id,
-            'text' => $this->name,
+            'text' => $this->furtherIcons . $this->name,
```

Comment:
It's a bit surprising that icon information is part of the name / text?

Reply:
That isn't possible an other way. jstree only offers ONE icon  and ONE text field with name "text". I cannot rename "text" because of jstree.